### DISCUSSION TITLE
Interoperability with AirDrop - DMA Consultation

### DISCUSSION THREAD

**Discussion by ghost** [localsend/localsend#2151] [Discussion (English, International)]
Hi everyone,

The European Commission [launched two consultations](https://digital-markets-act.ec.europa.eu/commission-seeks-feedback-measures-apple-should-take-ensure-interoperability-under-digital-markets-2024-12-19_en) to get feedback on the measures Apple should take to ensure interoperability under the [Digital Markets Act (DMA)](https://digital-markets-act.ec.europa.eu/index_en).

More specifically, [one of the two consultations is related to **iOS connectivity features**](https://digital-markets-act.ec.europa.eu/dma100203-consultation-proposed-measures-interoperability-between-apples-ios-operating-system-and_en), namely notifications, background execution, automatic Bluetooth audio switching, high-bandwidth peer-to-peer Wi-Fi connections, **AirDrop**, AirPlay, close-range wireless file transfers, media casting, proximity-triggered pairing, automatic Wi-Fi connection, and NFC functionality.

This could be of interest for LocalSend: the European Commission could force Apple to open up AirDrop, which could allow LocalSend to be interoperable with this technology.

I intend to submit feedback, and I invite you to do the same. The deadline is January 9, 2025.

**Comment by Tienisto** [comment]
Thanks @restuccia 
I will submit also.
